pkg/scheduler/capabilities/volumebinding/volume_binding.go

```diff
@@ -382,11 +412,11 @@ func New(ctx context.Context, plArgs runtime.Object, fh framework.Handle, fts fe
 	pvInformer := fh.SharedInformerFactory().Core().V1().PersistentVolumes()
 	storageClassInformer := fh.SharedInformerFactory().Storage().V1().StorageClasses()
 	csiNodeInformer := fh.SharedInformerFactory().Storage().V1().CSINodes()
-	var capacityCheck *CapacityCheck
+	var capacityCheck CapacityCheck
 	if options.ServerOpts.EnableCSIStorage {
-		capacityCheck = &CapacityCheck{
+		capacityCheck = CapacityCheck{
 			CSIDriverInformer:          fh.SharedInformerFactory().Storage().V1().CSIDrivers(),
-			CSIStorageCapacityInformer: fh.SharedInformerFactory().Storage().V1beta1().CSIStorageCapacities(),
```

Comment:
For being complitave being compatible with old k8s version, we should use discovery client to dynamically use v1 or v1beta.

Reply:
After you completely copying the volumebinding folder from Kubernetes version 1.30, Volcano needs to make modifications in the second commit to ensure compatibility with older versions of Kubernetes. This can be referenced from this commit：https://github.com/volcano-sh/volcano/pull/3295/commits/a6640730bb688add8721eb5c5395a87af0bd5297. It is essential to retain Volcano's existing APIs, such as adding v1beta1, etc.